packages/cdk8s-cli/README.md

```diff
@@ -52,6 +56,46 @@ objects:
 $ cdk8s import k8s
 ```
 
+### Import different versions of the k8s API
+
+By default, the command `cdk8s import k8s` will use the latest stable version of k8s objects.
+
+For example, when using k8s 1.17,  if you want to use `DeploymentSpec`, you would be importing it from [Deployment v1](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.17/#deploymentspec-v1-apps) by default. 
+
+In the default `v1`, the `revisionHistoryLimit`, which is the number of old ReplicaSets to retain, is set to 10 by default. You would have to explicitly set this to a higher number if you want to retain more.
+
+Let's say that you want the behavior of [extensions/v1beta1](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.17/#deploymentspec-v1beta1-extensions), which defaults to retaining _all_ old ReplicaSets.
+
+To do this, you could add the `--include` flag to your `cdk8s import`. Here is how that would look:
+
+```code
+cdk8s import k8s --language typescript --include io.k8s.api.extensions.v1beta1.Deployment
+```
+
+Here we are explicitly choosing to use the [Deployment extensions/v1beta1](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.17/#deployment-v1beta1-extensions) version, which includes the new default behavior that we want from `DeploymentSpec`
+
+### Import Customer Resource Definitions (CRDs)
+
+You can import CRDs from local files (`my_crd.yml`) or from a remote host (`https://raw.githubusercontent.com/jenkinsci/kubernetes-operator/master/deploy/crds/jenkins.io_jenkins_crd.yaml`)
+
+Just like other imports, this is how you do that:
+
+```code
+cdk8s import my_crd.yaml
+```
+
+By default, your imported CRD will be named by its `spec.names.kind`. If your CRD file contains multiple CRDs, this will create multiple imports, each named by its respective `spec.names.kind`.
+
+### Override default behavior of import naming
+
+You can also override the default behavior for `cdk8s` importing. This will be helpful to you if you would like to name your CRD imports differently from their kind. Here is how you do it:
+
+```code
+cdk8s import example:=my_crd.yaml
+```
+
+In this case, your import would be named `example`.
```

Comment:
Not accurate, right? This will only be used as a prefix, while the suffix is still going to be the CRD kind name.
Also worth mentioning how this plays out in each language.

Reply:
Mentioned: `Note: this is not yet supported for python.`